docs/guides/basics/authentication/index.md

```diff
@@ -316,6 +316,28 @@ Enabling the option `shouldAutoAuthSetup: true` further simplifies the sign-in e
 [View Sample Code][sample_coda_api]{ .md-button }
 
 
+### AWS Signautre Version 4
+
+Amazon Web Services (AWS) uses a a proprietary authentication scheme that requires each requests to be signed with a key and secret. Packs support this using the [`AWSAccessKey`][AWSAccessKey] authentication. You must specify which AWS service you are connecting to, since that is a required part of the signature. For example:
```

Comment:
AWS supports multiple forms of authentication.  This is one of them.  Might want to clarify the statement here.

Reply:
Good callout, done.